packages/router/src/adapters/subgraph/subgraph.ts

```diff
@@ -227,6 +218,14 @@ export const getActiveTransactions = async (_requestContext?: RequestContext): P
           }
 
           if (!receiving) {
+            // if not synced, cancel
+            if (!synced) {
+              return {
+                crosschainTx: sdkSenderTransactionToCrosschainTransaction(senderTx),
+                payload: {},
+                status: CrosschainTransactionStatus.ReceiverNotConfigured,
+              } as ActiveTransaction<"ReceiverNotConfigured">;
+            }
```

Comment:
This should really have a separate status that is handled that way. The subgraph being unsynced is not the same as the `ReceiverNotConfigured` status, which is otherwise only given to transactions where the `transaction.receivingChainId` is not configured (see [here](https://github.com/connext/nxtp/blob/b20e9b8ec68b21b4bd2ee1b836411373e6ceaab8/packages/router/src/adapters/subgraph/subgraph.ts#L131-L157))

Reply:
Agree, was too lazy to add another status.